kofta/src/vscode-webview/components/MicPermissionBanner.tsx

```diff
@@ -7,8 +8,21 @@ import { Button } from "./Button";
 interface MicPermissionBannerProps {}
 
 export const MicPermissionBanner: React.FC<MicPermissionBannerProps> = () => {
-  const { error } = useMicPermErrorStore();
+  const { error, set } = useMicPermErrorStore();
   const [count, setCount] = useState(0);
+  const history = useHistory();
+
+  useEffect(() => {
+    const removeListener = history.listen(() => {
+      // Dismiss banner when route changes.
+      set({ error: false });
+    });
+
+    return () => {
+      removeListener();
+    };
+  }, [history, set]);
```

Comment:
I don't want history.listen part, just the ability to dismiss

Reply:
Removed.